src/OpenLoco/Graphics/ImageIds.h

```diff
@@ -1015,6 +1015,8 @@ namespace OpenLoco::ImageIds
     constexpr uint32_t music_controls_next = 2385;
     constexpr uint32_t refit_cargo_button = 2386;
     constexpr uint32_t tab_vehicle_background = 2387;
+    constexpr uint32_t tab = 2387;
```

Comment:
this has the same number as the above `tab_vehicle_background`?

Reply:
Yeah need to look into this. I just noticed it out of place. I think tab_vehicle can go.